src/webparts/projectFilesViewer/components/models/IProjectFilesViewerState.ts

```diff
@@ -0,0 +1,3 @@
+export interface State{
```

Comment:
Change the name from `State` to `IProjectFilesViewerState`

Reply:
Done